tests/test_varname.py

```diff
@@ -382,7 +407,8 @@ def do(self):
             return 'I will do something'
 
     c = C()
-    assert c.iwill.do() == 'I will do something'
+    result = c.iwill.do()
+    assert result == 'I will do something'
```

Comment:
This innocent-looking change is what causes the loss of coverage. Making only this change on master has the same effect. Something very fishy is going on and I don't really want to understand it, it looks like more things need removing.